.ci/install_runtime.sh

```diff
@@ -186,20 +210,10 @@ if [ "$arch" == "aarch64" -a "${KATA_HYPERVISOR}" == "qemu" -a "${ENABLE_ARM64_U
 	sudo sed -i 's|#pflashes|pflashes|' "${runtime_config_path}"
 fi
 
-if [ "$TEE_TYPE" == "tdx" ]; then
+if [ "$TEE_TYPE" == "tdx" -a "${KATA_HYPERVISOR}" == "cloud-hypervisor" ]; then
         echo "Use tdx enabled guest config in ${runtime_config_path}"
         sudo sed -i -e 's/vmlinux.container/vmlinuz-tdx.container/' "${runtime_config_path}"
         sudo sed -i -e 's/^# confidential_guest/confidential_guest/' "${runtime_config_path}"
         sudo sed -i -e 's/^kernel_params = "\(.*\)"/kernel_params = "\1 force_tdx_guest tdx_disable_filter "/g' "${runtime_config_path}"
-
-        case "${KATA_HYPERVISOR}" in
-		"cloud-hypervisor")
-			sudo sed -i -e 's/^firmware = ".*"/firmware = "\/usr\/share\/td-shim\/final-pe.bin"/' "${runtime_config_path}"
-			;;
-
-		"qemu")
-			sudo sed -i -e 's/^firmware = ".*"/firmware = "\/usr\/share\/qemu\/OVMF.fd"/' "${runtime_config_path}"
-			sudo sed -i -e 's/^firmware_volume = ".*"/firmware_volume = "\/usr\/share\/qemu\/OVMF_VARS.fd"/' "${runtime_config_path}"
-			;;
-        esac
+	sudo sed -i -e 's/^firmware = ".*"/firmware = "\/usr\/share\/td-shim\/final-pe.bin"/' "${runtime_config_path}"
```

Comment:
Everything here can be dropped and we can do something similar to what you're doing with QEMU, as soon as https://github.com/kata-containers/tests/pull/5065 lands.

Reply:
Great, dropped that piece!